Source/Views/PageView.swift

```diff
@@ -53,7 +53,7 @@ class PageView: UIScrollView {
   weak var pageViewDelegate: PageViewDelegate?
 
   var hasZoomed: Bool {
-    return zoomScale != 1.0
+    return zoomScale != minimumZoomScale
```

Comment:
This was a bug that now comes into play since we're tweaking the minimumZoomScale.